dev/validate_bom__test.py

```diff
@@ -593,6 +632,44 @@ def wait_on_service(self, service_name, port=None, timeout=None):
         ResponseError('It appears that {0} failed'.format(service_name),
                       server='tunnel'))
 
+  def __wait_on_service_using_host(self, service_name, timeout=None):
+    service_config = self.__outbound_service_configs[service_name]
+    host = service_config['host']
+
+    timeout = timeout or self.options.test_service_startup_timeout
+    end_time = time.time() + timeout
+    logging.info('Waiting on "%s..."', service_name)
+
+    threadid = threading.current_thread().ident
+    logging.info('WaitOn polling %s from thread %x', service_name, threadid)
```

Comment:
does logging include the thread already? It's ok to leave given the concurrency alignment going through here.